app/controllers/passengers_controller.rb

```diff
@@ -48,8 +48,11 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
-      :wheelchair, :active, :permanent, :expiration, :note)
+    permitted_params = params.require(:passenger).permit(:name, :address, :email, :phone,
+        :wheelchair, :active, :permanent, :expiration, :note)
+    unless @current_user.admin?
+      permitted_params = permitted_params.except(:active, :permanent)
+    end
```

Comment:
It probably should return what you say, but at the moment, it returns the allowed params for non-admins, and `nil` for admins (because of implicit returns). You probably want an *explicit* `return`